### DISCUSSION TITLE
Daily 06/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#172] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Hoy estuve viendo la plataforma, ya está subido lo del crud con sqlite así que voy a trabajar sobre eso. También voy a ver de nuevo los videos que se mandaron al grupo, sobre todo el último y a tratar de pasarlo al login de nuestro proyecto. Sigo teniendo dificultades para comprender el back y esta semana no tuve mucho tiempo, así que el finde estaré también con este tema.

**Comment by BraianTroncoso** [comment]
Hoy estuve estudiando toda la mañana ya avanzando con el back, estuve leyendo documentacion y viendo el material del ISPC que dejaron. También tengo clases de ingles y voy a continuar con el proyecto despues o por la mña porque estuve con mucho trabajo y se me complicó un poco hoy.  Ofreci mi ayuda a quienes se le dificulten en el back y brinde material complementario.
Asi que todo finde voy a estar metiendole a esto básicamente.

**Comment by Valentino399** [comment]
Hoy estuve viendo los tipos primitivos en java y participe en la clase de ingles.
